Yoast/wordpress-seo
# Create an unindexed indexable when build fails

## Summary


This PR can be summarized in the following changelog entry:

* Prevents the indexation from continuing indefinitely.

## Test instructions

This PR can be tested by following these steps:

* Create a broken term in your database by deleting the corresponding entry in your `wp_term_taxonomy` table but not the `wp_terms` table.
* Run the indexation.
* It should complete and the notice should disappear.

## UI changes

## Documentation

## Quality assurance


Fixes #14972
